database/models/core.py

```diff
@@ -158,6 +169,49 @@ def __repr__(self):
         return f"Repo<{self.repoid}>"
 
 
+GITHUB_APP_INSTALLATION_DEFAULT_NAME = "codecov_app_installation"
+
+
+class GithubAppInstallation(CodecovBaseModel, MixinBaseClass):
```

Comment:
Is there an easy way to test this E2E to prevent something like what happened in API?

Reply:
Unsure. I don't think we have the infra in place...

I tested locally, if that's worth anything (upload, PR comment, statuses and exploring files in the UI), but it's very much a "trust me" scenario and we know it's easy to make mistakes.